src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx

```diff
@@ -87,43 +126,33 @@ const WorkPackageEditDetails: React.FC<Props> = ({ users, control, errors }) =>
             />
           </FormControl>
         </Grid>
-        <Grid item xs={12} md={2} sx={{ mt: 2, mr: 2 }}>
-          <FormControl fullWidth>
-            <FormLabel>Project Lead</FormLabel>
-            <Controller
-              name="projectLead"
-              control={control}
-              rules={{ required: true }}
-              render={({ field: { onChange, value } }) => (
-                <TextField select onChange={onChange} value={value} fullWidth>
-                  {users.map((t) => (
-                    <MenuItem key={t.userId} value={t.userId}>
-                      {fullNamePipe(t)}
-                    </MenuItem>
-                  ))}
-                </TextField>
-              )}
+        <Grid item xs={12} md={2} sx={{ mt: 2, mr: 29 }}>
+          <Grid item xs={12} md={8}>
```

Comment:
no need to put grid items inside each other. remove the embedded grid items, make md = 6, make width for NERAutocomplete like 90%, and then remove all the other styles with mt, sx, etc...

the way md/xs/lg and all of those work is as follows:

xs means extra small
md means medium
lg means large
etc..

the page will do whatever it is closest to, so if it is fullscreen and only xs and md are given, it will do whatever md says because the screen is closer to medium than extra small.

there are 12 columns in a page, and the number inside md is how many columns it will take up. so if we make md=6 for both, they will both be half the page (there is a spcing set at the top so it wont quite fit thpugh), so to make them not stick, we make the width of each ~90%. this will put a spacing between them because they take up 90% of their 6 columns and they are centered in those columns. 

after all of this, if something doesnt look right, then use mt because ur gonna need to add a little margin to the top